### ISSUE TITLE
Request: Stack total values

### ISSUE THREAD

**Issue by Kherae** [godfat/rest-gw2#12]
the materials storage stuff is useful, but it doesn't display the total value of stacks. adding this would make it easier to figure out where all our total wealth is.

**Comment by godfat** [comment]
Yes, I have thought about this, but as you can see, there's not much space left. Maybe tooltip? But that doesn't work well on mobile. Maybe just put it as the next line? But that's pretty messy.

Any idea how this could fit into the space?

**Comment by Kherae** [comment]
Append to price line in parentheses, is my suggestion
I'm not sure how it would look on mobile, as I haven't one. but on a web browser, that would look neatest

**Comment by godfat** [comment]
I'll try how that looks, thanks!

**Comment by godfat** [comment]
It doesn't look good in my opinion, but I think we could keep it for now before I have a better idea... Can't dedicate a lot of time to figure this out for now.

![screen shot 2018-05-04 at 22 19 50](https://user-images.githubusercontent.com/10416/39632993-51f709ee-4fe9-11e8-9f13-bedb968be628.png)

I'll leave this open until I am satisfied with the result.

**Comment by Kherae** [comment]
yeah, it doesn't look right. and I noticed that when I hovered over one entry, it ran onto another.
could put it on the next line and adjust the alignment. might have just enough room. still, having the functionality, for now at least, is useful.
